### PR TITLE
feat(#1979) Sonar Qube Complaints

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
Fix all Sonar Tube Complaints.

Closes: #1979

### CODE REVIEW

File: eo-runtime/src/main/java/org/eolang/BytesOf.java [hunk 1]
```diff
@@ -142,7 +142,7 @@ public Bytes shift(final int bits) {
                 } else {
                     byte dst = (byte) (bytes[source] << mod);
                     if (source + 1 < bytes.length) {
-                        dst |= bytes[source + 1] >>> (Byte.SIZE - mod) & carry & 0xFF;
+                        dst |= bytes[source + 1] >>> (Byte.SIZE - mod) & (carry & 0xFF) & 0xFF;
```
Comment (mximp) [inline eo-runtime/src/main/java/org/eolang/BytesOf.java:145]: @volodya-lombrozo I believe just adding braces would do the trick. No need in second `& 0xFF`

Comment (volodya-lombrozo) [reply eo-runtime/src/main/java/org/eolang/BytesOf.java]: @mximp You right. Thanks.

File: eo-runtime/src/main/java/org/eolang/BytesOf.java [hunk 2]
```diff
@@ -156,7 +156,7 @@ public Bytes shift(final int bits) {
                 } else {
                     byte dst = (byte) ((0xff & bytes[source]) >>> mod);
                     if (source - 1 >= 0) {
-                        dst |= bytes[source - 1] << (Byte.SIZE - mod) & carry & 0xFF;
+                        dst |= bytes[source - 1] << (Byte.SIZE - mod) & (carry & 0xFF) & 0xFF;
```
[the remark this answers is not in the document]
Comment (volodya-lombrozo) [reply eo-runtime/src/main/java/org/eolang/BytesOf.java]: @mximp You right. Thanks.

[the remark this answers is not in the document]
Comment (volodya-lombrozo) [reply eo-runtime/src/main/java/org/eolang/BytesOf.java]: @mximp You right. Thanks.

**Review by mximp — APPROVED** [review]

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you merge that changes, please?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**